### DISCUSSION TITLE
Helllooooo????

### DISCUSSION THREAD

**Discussion by MrConqueror** [nikhilbadyal/docker-py-revanced#440] [Q&A] [answered]
https://github.com/nikhilbadyal/docker-py-revanced/assets/102837820/24da7693-7eba-49ca-a6d5-5fb97b1de36e
It takes me to browser instead in App.
Is anyone here have the same issue?

**Comment by nikhilbadyal** [comment]
[Google it.](https://www.google.com/search?q=open+link+with+native+app+instead+of+browser&oq=open+link+with+native+app+instead+of+browser&gs_lcrp=EgZjaHJvbWUyBggAEEUYOdIBCTEyMzE2ajBqOagCALACAA&sourceid=chrome-mobile&ie=UTF-8)

**Reply by MrConqueror** [reply]
I can't find the solution there, it still says "see this post" in chrome or reddit when i already have reddit app of yours.

**Reply by IMXEren** [reply]
Reddit > App Info > Open supported links > Add links.

**Reply by MrConqueror** [reply]
![Screenshot_2024-01-16-17-55-50-365_com miui securitycenter](https://github.com/nikhilbadyal/docker-py-revanced/assets/102837820/8cf04657-dffc-4ed8-b0fc-0f560e3a11e7)
where???!! I don't have that kind of feature in my phone

**Reply by MrConqueror** [reply]
Uploading Screenrecorder-2024-01-16-18-01-34-798.mp4…

**Reply by nikhilbadyal** [reply]
Click on **Clear Default** in the last.

**Reply by MrConqueror** [reply]
it just says no defaults set🤷

**Reply by nikhilbadyal** [reply]
Share video while clicking on it.

**Reply by MrConqueror** [reply]
https://github.com/nikhilbadyal/docker-py-revanced/assets/102837820/e81b188c-76ee-4817-930c-d219187747c3

**Reply by MrConqueror** [reply, marked as answer]
https://github.com/nikhilbadyal/docker-py-revanced/assets/102837820/23c165c2-7b0b-462b-9a42-a0ad60eb6592

my problem has been solved i uninstall the app out of frustration but i installed it again.
